CMakeLists.txt

```diff
@@ -1,35 +1,48 @@
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.17)
+project(kivi)
```

Comment:
Add additional information: `VERSION 0.01`, `HOMEPAGE_URL`, etc.

Check [the docs](https://cmake.org/cmake/help/latest/command/project.html) for more details.